src/index.ts

```diff
@@ -1,55 +1,51 @@
 import axios from 'axios';
 import Ain from '@ainblockchain/ain-js';
-import Assets from './assets';
+import { Account } from '@ainblockchain/ain-js/lib/types';
+import Asset from './asset';
 import Discord from './discord';
-import EventManager from './event';
+import Event from './event';
 import {
   AINFT_SERVER_ENDPOINT,
   AIN_BLOCKCHAIN_CHAINID,
   AIN_BLOCKCHAIN_ENDPOINT,
 } from './constants';
-import { Account } from './types';
 
 export default class AinftJs {
   private baseUrl: string;
   public accessAccount: Account;
-  public assets: Assets;
+  public asset: Asset;
   public discord: Discord;
-  public eventManager: EventManager;
+  public event: Event;
   public ain: Ain;
 
-  constructor(baseUrl = AINFT_SERVER_ENDPOINT, accessAccount: Account) {
+  constructor(baseUrl = AINFT_SERVER_ENDPOINT, accessAccountPrivateKey: string) {
     this.baseUrl = baseUrl;
     // NOTE(liayoo): added to avoid an uninitialized error
-    this.accessAccount = accessAccount;
+    this.accessAccount = {
```

Comment:
How about removing this? Already, it's removed in another class.

Reply:
Removed the class variable and added a getAccessAccount function instead that returns the ain wallet's current defaultAccount.